kubernetes/loculus/values.yaml

```diff
@@ -1451,7 +1450,7 @@ insecureCookies: false
 bannerMessage: "This is a demonstration environment. It may contain non-accurate test data and should not be used for real-world applications. Data will be deleted regularly."
 additionalHeadHTML: '<script defer data-domain="loculus.org" src="https://plausible.io/js/script.js"></script>'
 images:
-  lapisSilo: "ghcr.io/genspectrum/lapis-silo:0.2.23"
+  lapisSilo: "ghcr.io/genspectrum/lapis-silo:0.3.0"
   lapis: "ghcr.io/genspectrum/lapis:0.3.5"
```

Comment:
You should also update LAPIS (check the README there for which version you need for SILO 0.3 - I think it's 0.3.7)